jdbc/deployment/src/test/resources/custom_application.properties

```diff
@@ -0,0 +1,7 @@
+%test.quarkus.datasource.db-kind=h2
+%test.quarkus.datasource.jdbc.url=jdbc:h2:tcp://localhost/mem:test;DB_CLOSE_DELAY=-1;INIT=RUNSCRIPT FROM 'src/test/resources/custom_import.sql'
+%test.quarkus.datasource.username=sa
+%test.quarkus.datasource.password=sa
+%test.quarkus.jdbc-config.table=custom_config
+%test.quarkus.jdbc-config.key-column=a
+%test.quarkus.jdbc-config.value-column=b
```

Comment:
I think these should be grouped in `quarkus.config.source.jdbc` namespace and not be part of the `quarkus` root namespace. So: 

- `quarkus.config.source.jdbc.table`
- `quarkus.config.source.jdbc.key`
- `quarkus.config.source.jdbc.value`